<!DOCTYPE html> <html lang="en"> <head> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title> Sohrab Salehi </title> <meta name="author" content="Sohrab Salehi "> <meta name="description" content="Personal website for Sohrab Salehi, a computational biologist at Memorial Sloan Kettering Cancer Center. "> <meta name="keywords" content="cancer evolution, causal inference, drug-resistance, metastasis, portfolio-website"> <link rel="stylesheet" href="/assets/css/bootstrap.min.css?a4b3f509e79c54a512b890d73235ef04"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"> <link defer rel="stylesheet" href="/assets/css/academicons.min.css?f0b7046b84e425c55f3463ac249818f5"> <link defer rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons&amp;display=swap"> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-github.css?591dab5a4e56573bf4ef7fd332894c99" media="" id="highlight_theme_light"> <link rel="shortcut icon" href="data:image/svg+xml,&lt;svg%20xmlns=%22http://www.w3.org/2000/svg%22%20viewBox=%220%200%20100%20100%22&gt;&lt;text%20y=%22.9em%22%20font-size=%2290%22&gt;%E2%9A%9B%EF%B8%8F&lt;/text&gt;&lt;/svg&gt;"> <link rel="stylesheet" href="/assets/css/main.css?d41d8cd98f00b204e9800998ecf8427e"> <link rel="canonical" href="https://sohrabsa.github.io/"> <script src="/assets/js/theme.js?9a0c749ec5240d9cda97bc72359a72c0"></script> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-native.css?5847e5ed4a4568527aa6cfab446049ca" media="none" id="highlight_theme_dark"> <script>initTheme();</script> </head> <body class="fixed-top-nav "> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top" role="navigation"> <div class="container"> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item active"> <a class="nav-link" href="/">About <span class="sr-only">(current)</span> </a> </li> <li class="nav-item "> <a class="nav-link" href="/publications/">Publications </a> </li> <li class="toggle-container"> <button id="light-toggle" title="Change theme"> <i class="ti ti-sun-moon" id="light-toggle-system"></i> <i class="ti ti-moon-filled" id="light-toggle-dark"></i> <i class="ti ti-sun-filled" id="light-toggle-light"></i> </button> </li> </ul> </div> </div> </nav> <progress id="progress" value="0"> <div class="progress-container"> <span class="progress-bar"></span> </div> </progress> </header> <div class="container mt-5" role="main"> <div class="post"> <header class="post-header"> <h1 class="post-title"> <span class="font-weight-bold">Sohrab Salehi</span> </h1> <p class="desc"></p> </header> <article> <div class="profile float-right"> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/prof_pic-480.webp 480w,/assets/img/prof_pic-800.webp 800w,/assets/img/prof_pic-1400.webp 1400w," sizes="(min-width: 930px) 270.0px, (min-width: 576px) 30vw, 95vw" type="image/webp"> <img src="/assets/img/prof_pic.jpg?4c99ad1181157d6669c5fae62f7d816c" class="img-fluid z-depth-1 rounded" width="100%" height="auto" alt="prof_pic.jpg" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </source></picture> </figure> <div class="more-info"> <p>Memorial Sloan Kettering Cancer Center</p> <p>321 East 61st Street</p> <p>New York, NY 10065</p> </div> </div> <div class="clearfix"> <p>I am a postdoctoral research fellow at Memorial Sloan Kettering Cancer Center in the Department of Epidemiology and Biostatistics under Drs. <a href="https://www.mskcc.org/research-areas/labs/sohrab-shah" rel="external nofollow noopener" target="_blank">Sohrab P. Shah</a>, <a href="https://www.mskcc.org/research-areas/labs/charles-rudin" rel="external nofollow noopener" target="_blank">Charles M. Rudin</a>, and a postdoctoral research scientist at the Irving Institute for Cancer Dynamics at Columbia Univeristy under Dr. <a href="https://www.cs.columbia.edu/~blei/" rel="external nofollow noopener" target="_blank">David M. Blei</a>. My current research focuses on developing causal inference methods to understand mechanisms of drug-resistance and metastasis in human cancers at a single cell level. I did my PhD in Bioinformatics at University of British Columbia (UBC) under Dr. <a href="https://www.stat.ubc.ca/~bouchard/" rel="external nofollow noopener" target="_blank">Alexandre Bouchard-Côté</a>. My PhD research focused on developing Bayesian models to quantify the evolutionary dynamics and fitness of human cancers from single cell whole genome sequencing data.</p> </div> <h2> <a href="/publications/" style="color: inherit">Selected Publications</a> </h2> <div class="publications"> <ol class="bibliography"> <li> <div class="row"> <div id="10_24072_pcjournal_292" class="col-sm-10"> <div class="title">Cancer phylogenetic tree inference at scale from 1000s of single cell genomes</div> <div class="author"> <em>Sohrab Salehi</em>, Fatemeh Dorri, Kevin Chern, Farhia Kabeer, and <span class="more-authors" title="click to view 11 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '11 more authors' ? 'Nicole Rusk, Tyler Funnell, Marc J. Williams, Daniel Lai, Mirela Andronescu, Kieran R. Campbell, Andrew McPherson, Samuel Aparicio, Andrew Roth, Sohrab P. Shah, Alexandre Bouchard-Côté' : '11 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '5'); ">11 more authors</span> </div> <div class="periodical"> 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://peercommunityjournal.org/articles/10.24072/pcjournal.292/" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> </div> <div class="abstract hidden"> <p>A new generation of scalable single cell whole genome sequencing (scWGS) methods allows unprecedented high resolution measurement of the evolutionary dynamics of cancer cell populations. Phylogenetic reconstruction is central to identifying sub-populations and distinguishing the mutational processes that gave rise to them. Existing phylogenetic tree building models do not scale to the tens of thousands of high resolution genomes achievable with current scWGS methods. We constructed a phylogenetic model and associated Bayesian inference procedure, sitka, specifically for scWGS data. The method is based on a novel phylogenetic encoding of copy number (CN) data, the sitka transformation, that simplifies the site dependencies induced by rearrangements while still forming a sound foundation to phylogenetic inference. The sitka transformation allows us to design novel scalable Markov chain Monte Carlo (MCMC) algorithms. Moreover, we introduce a novel point mutation calling method that incorporates the CN data and the underlying phylogenetic tree to overcome the low per-cell coverage of scWGS. We demonstrate our method on three single cell datasets, including a novel PDX series, and analyse the topological properties of the inferred trees. Sitka is freely available at ‘https://github.com/UBC-Stat-ML/sitkatree.git‘.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">10_24072_pcjournal_292</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Salehi, Sohrab and Dorri, Fatemeh and Chern, Kevin and Kabeer, Farhia and Rusk, Nicole and Funnell, Tyler and Williams, Marc J. and Lai, Daniel and Andronescu, Mirela and Campbell, Kieran R. and McPherson, Andrew and Aparicio, Samuel and Roth, Andrew and Shah, Sohrab P. and Bouchard-C\ot\'e, Alexandre}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Cancer phylogenetic tree inference at scale from 1000s of single cell genomes}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Peer Community Journal}</span><span class="p">,</span>
  <span class="na">eid</span> <span class="p">=</span> <span class="s">{e63}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Peer Community In}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{3}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2023}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.24072/pcjournal.292}</span><span class="p">,</span>
  <span class="na">language</span> <span class="p">=</span> <span class="s">{en}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://peercommunityjournal.org/articles/10.24072/pcjournal.292/}</span><span class="p">,</span>
  <span class="na">type</span> <span class="p">=</span> <span class="s">{first_author}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div id="genomic_variation_funnell" class="col-sm-10"> <div class="title">Single-cell genomic variation induced by mutational processes in cancer</div> <div class="author"> Tyler Funnell, Ciara H. O’Flanagan, Marc J. Williams, Andrew McPherson, and <span class="more-authors" title="click to view 116 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '116 more authors' ? 'Steven McKinney, Farhia Kabeer, Hakwoo Lee, Sohrab Salehi, Ignacio Vázquez-Garcı́a, Hongyu Shi, Emily Leventhal, Tehmina Masud, Peter Eirew, Damian Yap, Allen W. Zhang, Jamie L. P. Lim, Beixi Wang, Jazmine Brimhall, Justina Biele, Jerome Ting, Vinci Au, Michael Van Vliet, Yi Fei Liu, Sean Beatty, Daniel Lai, Jenifer Pham, Diljot Grewal, Douglas Abrams, Eliyahu Havasov, Samantha Leung, Viktoria Bojilova, Richard A. Moore, Nicole Rusk, Florian Uhlitz, Nicholas Ceglia, Adam C. Weiner, Elena Zaikova, J. Maxwell Douglas, Dmitriy Zamarin, Britta Weigelt, Sarah H. Kim, Arnaud Da Cruz Paula, Jorge S. Reis-Filho, Spencer D. Martin, Yangguang Li, Hong Xu, Teresa Ruiz Algara, So Ra Lee, Viviana Cerda Llanos, David G. Huntsman, Jessica N. McAlpine, Gregory J. Hannon, Georgia Battistoni, Dario Bressan, Ian G. Cannell, Hannah Casbolt, Cristina Jauset, Tatjana Kovačević, Claire M. Mulvey, Fiona Nugent, Marta Paez Ribes, Isabella Pearson, Fatime Qosaj, Kirsty Sawicka, Sophia A. Wild, Elena Williams, Emma Laks, Austin Smith, Andrew Roth, Shankar Balasubramanian, Maximilian Lee, Bernd Bodenmiller, Marcel Burger, Laura Kuett, Sandra Tietscher, Jonas Windhager, Edward S. Boyden, Shahar Alon, Yi Cui, Amauche Emenari, Daniel R. Goodwin, Emmanouil D. Karagiannis, Anubhav Sinha, Asmamaw T. Wassie, Carlos Caldas, Alejandra Bruna, Maurizio Callari, Wendy Greenwood, Giulia Lerda, Yaniv Eyal-Lubling, Oscar M. Rueda, Abigail Shea, Owen Harris, Robby Becker, Flaminia Grimaldo, Suvi Harris, Sara Lisa Vogl, Johanna A. Joyce, Spencer S. Watson, Simon Tavare, Khanh N. Dinh, Eyal Fisher, Russell Kunes, Nicholas A. Walton, Mohammed Al Sa’d, Nick Chornay, Ali Dariush, Eduardo A. González-Solares, Carlos González-Fernández, Aybüke Küpcü Yoldaş, Neil Miller, Xiaowei Zhuang, Jean Fan, Hsuan Lee, Leonardo A. Sepúlveda, Chenglong Xia, Pu Zheng, Sohrab P. Shah, Samuel Aparicio, IMAXT Consortium' : '116 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '5'); ">116 more authors</span> </div> <div class="periodical"> <em>Nature</em>, 2022 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://doi.org/10.1038/s41586-022-05249-0" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> </div> <div class="abstract hidden"> <p>How cell-to-cell copy number alterations that underpin genomic instability1 in human cancers drive genomic and phenotypic variation, and consequently the evolution of cancer2, remains understudied. Here, by applying scaled single-cell whole-genome sequencing3 to wild-type, TP53-deficient and TP53-deficient;BRCA1-deficient or TP53-deficient;BRCA2-deficient mammary epithelial cells (13,818 genomes), and to primary triple-negative breast cancer (TNBC) and high-grade serous ovarian cancer (HGSC) cells (22,057 genomes), we identify three distinct ‘foreground’mutational patterns that are defined by cell-to-cell structural variation. Cell- and clone-specific high-level amplifications, parallel haplotype-specific copy number alterations and copy number segment length variation (serrate structural variations) had measurable phenotypic and evolutionary consequences. In TNBC and HGSC, clone-specific high-level amplifications in known oncogenes were highly prevalent in tumours bearing fold-back inversions, relative to tumours with homologous recombination deficiency, and were associated with increased clone-to-clone phenotypic variation. Parallel haplotype-specific alterations were also commonly observed, leading to phylogenetic evolutionary diversity and clone-specific mono-allelic expression. Serrate variants were increased in tumours with fold-back inversions and were highly correlated with increased genomic diversity of cellular populations. Together, our findings show that cell-to-cell structural variation contributes to the origins of phenotypic and evolutionary diversity in TNBC and HGSC, and provide insight into the genomic and mutational states of individual cancer cells.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">genomic_variation_funnell</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Funnell, Tyler and O'Flanagan, Ciara H. and Williams, Marc J. and McPherson, Andrew and McKinney, Steven and Kabeer, Farhia and Lee, Hakwoo and Salehi, Sohrab and V{\'a}zquez-Garc{\'\i}a, Ignacio and Shi, Hongyu and Leventhal, Emily and Masud, Tehmina and Eirew, Peter and Yap, Damian and Zhang, Allen W. and Lim, Jamie L. P. and Wang, Beixi and Brimhall, Jazmine and Biele, Justina and Ting, Jerome and Au, Vinci and Van Vliet, Michael and Liu, Yi Fei and Beatty, Sean and Lai, Daniel and Pham, Jenifer and Grewal, Diljot and Abrams, Douglas and Havasov, Eliyahu and Leung, Samantha and Bojilova, Viktoria and Moore, Richard A. and Rusk, Nicole and Uhlitz, Florian and Ceglia, Nicholas and Weiner, Adam C. and Zaikova, Elena and Douglas, J. Maxwell and Zamarin, Dmitriy and Weigelt, Britta and Kim, Sarah H. and Da Cruz Paula, Arnaud and Reis-Filho, Jorge S. and Martin, Spencer D. and Li, Yangguang and Xu, Hong and de Algara, Teresa Ruiz and Lee, So Ra and Llanos, Viviana Cerda and Huntsman, David G. and McAlpine, Jessica N. and Hannon, Gregory J. and Battistoni, Georgia and Bressan, Dario and Cannell, Ian G. and Casbolt, Hannah and Jauset, Cristina and Kova{\v c}evi{\'c}, Tatjana and Mulvey, Claire M. and Nugent, Fiona and Ribes, Marta Paez and Pearson, Isabella and Qosaj, Fatime and Sawicka, Kirsty and Wild, Sophia A. and Williams, Elena and Laks, Emma and Smith, Austin and Roth, Andrew and Balasubramanian, Shankar and Lee, Maximilian and Bodenmiller, Bernd and Burger, Marcel and Kuett, Laura and Tietscher, Sandra and Windhager, Jonas and Boyden, Edward S. and Alon, Shahar and Cui, Yi and Emenari, Amauche and Goodwin, Daniel R. and Karagiannis, Emmanouil D. and Sinha, Anubhav and Wassie, Asmamaw T. and Caldas, Carlos and Bruna, Alejandra and Callari, Maurizio and Greenwood, Wendy and Lerda, Giulia and Eyal-Lubling, Yaniv and Rueda, Oscar M. and Shea, Abigail and Harris, Owen and Becker, Robby and Grimaldo, Flaminia and Harris, Suvi and Vogl, Sara Lisa and Joyce, Johanna A. and Watson, Spencer S. and Tavare, Simon and Dinh, Khanh N. and Fisher, Eyal and Kunes, Russell and Walton, Nicholas A. and Al Sa'd, Mohammed and Chornay, Nick and Dariush, Ali and Gonz{\'a}lez-Solares, Eduardo A. and Gonz{\'a}lez-Fern{\'a}ndez, Carlos and Yolda{\c s}, Ayb{\"u}ke K{\"u}pc{\"u} and Miller, Neil and Zhuang, Xiaowei and Fan, Jean and Lee, Hsuan and Sep{\'u}lveda, Leonardo A. and Xia, Chenglong and Zheng, Pu and Shah, Sohrab P. and Aparicio, Samuel and Consortium, IMAXT}</span><span class="p">,</span>
  <span class="na">date</span> <span class="p">=</span> <span class="s">{2022/12/01}</span><span class="p">,</span>
  <span class="na">date-added</span> <span class="p">=</span> <span class="s">{2024-09-02 18:18:42 -0400}</span><span class="p">,</span>
  <span class="na">date-modified</span> <span class="p">=</span> <span class="s">{2024-09-02 18:18:42 -0400}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1038/s41586-022-05249-0}</span><span class="p">,</span>
  <span class="na">id</span> <span class="p">=</span> <span class="s">{Funnell2022}</span><span class="p">,</span>
  <span class="na">isbn</span> <span class="p">=</span> <span class="s">{1476-4687}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Nature}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{7938}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{106--115}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Single-cell genomic variation induced by mutational processes in cancer}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://doi.org/10.1038/s41586-022-05249-0}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{612}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2022}</span><span class="p">,</span>
  <span class="na">bdsk-url-1</span> <span class="p">=</span> <span class="s">{https://doi.org/10.1038/s41586-022-05249-0}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div id="salehi_clonal_2021" class="col-sm-10"> <div class="title">Clonal fitness inferred from time-series modelling of single-cell cancer genomes</div> <div class="author"> <em>Sohrab Salehi</em>, Farhia Kabeer, Nicholas Ceglia, Mirela Andronescu, and <span class="more-authors" title="click to view 104 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '104 more authors' ? 'Marc J. Williams, Kieran R. Campbell, Tehmina Masud, Beixi Wang, Justina Biele, Jazmine Brimhall, David Gee, Hakwoo Lee, Jerome Ting, Allen W. Zhang, Hoa Tran, Ciara O’Flanagan, Fatemeh Dorri, Nicole Rusk, Teresa Ruiz Algara, So Ra Lee, Brian Yu Chieh Cheng, Peter Eirew, Takako Kono, Jenifer Pham, Diljot Grewal, Daniel Lai, Richard Moore, Andrew J. Mungall, Marco A. Marra, Gregory J. Hannon, Giorgia Battistoni, Dario Bressan, Ian Gordon Cannell, Hannah Casbolt, Atefeh Fatemi, Cristina Jauset, Tatjana Kovačević, Claire M. Mulvey, Fiona Nugent, Marta Paez Ribes, Isabella Pearsall, Fatime Qosaj, Kirsty Sawicka, Sophia A. Wild, Elena Williams, Emma Laks, Yangguang Li, Ciara H. O’Flanagan, Austin Smith, Teresa Ruiz, Daniel Lai, Andrew Roth, Shankar Balasubramanian, Maximillian Lee, Bernd Bodenmiller, Marcel Burger, Laura Kuett, Sandra Tietscher, Jonas Windhager, Edward S. Boyden, Shahar Alon, Yi Cui, Amauche Emenari, Dan Goodwin, Emmanouil D. Karagiannis, Anubhav Sinha, Asmamaw T. Wassie, Carlos Caldas, Alejandra Bruna, Maurizio Callari, Wendy Greenwood, Giulia Lerda, Yaniv Eyal-Lubling, Oscar M. Rueda, Abigail Shea, Owen Harris, Robby Becker, Flaminia Grimaldi, Suvi Harris, Sara Lisa Vogl, Joanna Weselak, Johanna A. Joyce, Spencer S. Watson, Ignacio Vázquez-Garćıa, Simon Tavaré, Khanh N. Dinh, Eyal Fisher, Russell Kunes, Nicholas A. Walton, Mohammad Al Sa’d, Nick Chornay, Ali Dariush, Eduardo A. González-Solares, Carlos González-Fernández, Aybüke Küpcü Yoldas, Neil Millar, Tristan Whitmarsh, Xiaowei Zhuang, Jean Fan, Hsuan Lee, Leonardo A. Sepúlveda, Chenglong Xia, Pu Zheng, Andrew McPherson, Alexandre Bouchard-Côté, Samuel Aparicio, Sohrab P. Shah, IMAXT Consortium' : '104 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '5'); ">104 more authors</span> </div> <div class="periodical"> Jul 2021 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.nature.com/articles/s41586-021-03648-3" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> </div> <div class="abstract hidden"> <p>Progress in defining genomic fitness landscapes in cancer, especially those defined by copy number alterations (CNAs), has been impeded by lack of time-series single-cell sampling of polyclonal populations and temporal statistical models1–7. Here we generated 42,000 genomes from multi-year time-series single-cell whole-genome sequencing of breast epithelium and primary triple-negative breast cancer (TNBC) patient-derived xenografts (PDXs), revealing the nature of CNA-defined clonal fitness dynamics induced by TP53 mutation and cisplatin chemotherapy. Using a new Wright–Fisher population genetics model8,9 to infer clonal fitness, we found that TP53 mutation alters the fitness landscape, reproducibly distributing fitness over a larger number of clones associated with distinct CNAs. Furthermore, in TNBC PDX models with mutated TP53, inferred fitness coefficients from CNA-based genotypes accurately forecast experimentally enforced clonal competition dynamics. Drug treatment in three long-term serially passaged TNBC PDXs resulted in cisplatin-resistant clones emerging from low-fitness phylogenetic lineages in the untreated setting. Conversely, high-fitness clones from treatment-naive controls were eradicated, signalling an inversion of the fitness landscape. Finally, upon release of drug, selection pressure dynamics were reversed, indicating a fitness cost of treatment resistance. Together, our findings define clonal fitness linked to both CNA and therapeutic resistance in polyclonal tumours.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">salehi_clonal_2021</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Clonal fitness inferred from time-series modelling of single-cell cancer genomes}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{595}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{1476-4687}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://doi.org/10.1038/s41586-021-03648-3}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1038/s41586-021-03648-3}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{7868}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Nature}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Salehi, Sohrab and Kabeer, Farhia and Ceglia, Nicholas and Andronescu, Mirela and Williams, Marc J. and Campbell, Kieran R. and Masud, Tehmina and Wang, Beixi and Biele, Justina and Brimhall, Jazmine and Gee, David and Lee, Hakwoo and Ting, Jerome and Zhang, Allen W. and Tran, Hoa and O’Flanagan, Ciara and Dorri, Fatemeh and Rusk, Nicole and de Algara, Teresa Ruiz and Lee, So Ra and Cheng, Brian Yu Chieh and Eirew, Peter and Kono, Takako and Pham, Jenifer and Grewal, Diljot and Lai, Daniel and Moore, Richard and Mungall, Andrew J. and Marra, Marco A. and Hannon, Gregory J. and Battistoni, Giorgia and Bressan, Dario and Cannell, Ian Gordon and Casbolt, Hannah and Fatemi, Atefeh and Jauset, Cristina and Kovačević, Tatjana and Mulvey, Claire M. and Nugent, Fiona and Ribes, Marta Paez and Pearsall, Isabella and Qosaj, Fatime and Sawicka, Kirsty and Wild, Sophia A. and Williams, Elena and Laks, Emma and Li, Yangguang and O’Flanagan, Ciara H. and Smith, Austin and Ruiz, Teresa and Lai, Daniel and Roth, Andrew and Balasubramanian, Shankar and Lee, Maximillian and Bodenmiller, Bernd and Burger, Marcel and Kuett, Laura and Tietscher, Sandra and Windhager, Jonas and Boyden, Edward S. and Alon, Shahar and Cui, Yi and Emenari, Amauche and Goodwin, Dan and Karagiannis, Emmanouil D. and Sinha, Anubhav and Wassie, Asmamaw T. and Caldas, Carlos and Bruna, Alejandra and Callari, Maurizio and Greenwood, Wendy and Lerda, Giulia and Eyal-Lubling, Yaniv and Rueda, Oscar M. and Shea, Abigail and Harris, Owen and Becker, Robby and Grimaldi, Flaminia and Harris, Suvi and Vogl, Sara Lisa and Weselak, Joanna and Joyce, Johanna A. and Watson, Spencer S. and Vázquez-Garćıa, Ignacio and Tavaré, Simon and Dinh, Khanh N. and Fisher, Eyal and Kunes, Russell and Walton, Nicholas A. and Sa’d, Mohammad Al and Chornay, Nick and Dariush, Ali and González-Solares, Eduardo A. and González-Fernández, Carlos and Yoldas, Aybüke Küpcü and Millar, Neil and Whitmarsh, Tristan and Zhuang, Xiaowei and Fan, Jean and Lee, Hsuan and Sepúlveda, Leonardo A. and Xia, Chenglong and Zheng, Pu and McPherson, Andrew and Bouchard-Côté, Alexandre and Aparicio, Samuel and Shah, Sohrab P. and {IMAXT Consortium}}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">jul</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2021}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{585--590}</span><span class="p">,</span>
  <span class="na">type</span> <span class="p">=</span> <span class="s">{first_author}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div id="salehi_ddclone_2017" class="col-sm-10"> <div class="title">ddClone: joint statistical inference of clonal populations from single cell and bulk tumour sequencing data</div> <div class="author"> <em>Sohrab Salehi</em>, Adi Steif, Andrew Roth, Samuel Aparicio, and <span class="more-authors" title="click to view 2 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '2 more authors' ? 'Alexandre Bouchard-Côté, Sohrab P. Shah' : '2 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '5'); ">2 more authors</span> </div> <div class="periodical"> Mar 2017 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-017-1169-3" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> </div> <div class="abstract hidden"> <p>Next-generation sequencing (NGS) of bulk tumour tissue can identify constituent cell populations in cancers and measure their abundance. This requires computational deconvolution of allelic counts from somatic mutations, which may be incapable of fully resolving the underlying population structure. Single cell sequencing (SCS) is a more direct method, although its replacement of NGS is impeded by technical noise and sampling limitations. We propose ddClone, which analytically integrates NGS and SCS data, leveraging their complementary attributes through joint statistical inference. We show on real and simulated datasets that ddClone produces more accurate results than can be achieved by either method alone.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">salehi_ddclone_2017</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{{ddClone}: joint statistical inference of clonal populations from single cell and bulk tumour sequencing data}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{18}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{1474-760X}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://doi.org/10.1186/s13059-017-1169-3}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1186/s13059-017-1169-3}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{1}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Genome Biology}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Salehi, Sohrab and Steif, Adi and Roth, Andrew and Aparicio, Samuel and Bouchard-Côté, Alexandre and Shah, Sohrab P.}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">mar</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2017}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{44}</span><span class="p">,</span>
  <span class="na">type</span> <span class="p">=</span> <span class="s">{first_author}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> </ol> </div> </article> </div> </div> <footer class="fixed-bottom" role="contentinfo"> <div class="container mt-0"> © Copyright 2024 Sohrab Salehi . Powered by <a href="https://jekyllrb.com/" target="_blank" rel="external nofollow noopener">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio" rel="external nofollow noopener" target="_blank">al-folio</a> theme. Hosted by <a href="https://pages.github.com/" target="_blank" rel="external nofollow noopener">GitHub Pages</a>. </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="/assets/js/bootstrap.bundle.min.js"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@5.0.0/imagesloaded.pkgd.min.js" integrity="sha256-htrLFfZJ6v5udOG+3kNLINIKh2gvoKqwEhHYfTTMICc=" crossorigin="anonymous"></script> <script defer src="/assets/js/masonry.js" type="text/javascript"></script> <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.1.0/dist/medium-zoom.min.js" integrity="sha256-ZgMyDAIYDYGxbcpJcfUnYwNevG/xi9OHKaR/8GK+jWc=" crossorigin="anonymous"></script> <script defer src="/assets/js/zoom.js?85ddb88934d28b74e78031fd54cf8308"></script> <script src="/assets/js/no_defer.js?2781658a0a2b13ed609542042a859126"></script> <script defer src="/assets/js/common.js?e0514a05c5c95ac1a93a8dfd5249b92e"></script> <script defer src="/assets/js/copy_code.js?12775fdf7f95e901d7119054556e495f" type="text/javascript"></script> <script defer src="/assets/js/jupyter_new_tab.js?d9f17b6adc2311cbabd747f4538bb15f"></script> <script async src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script> <script async src="https://badge.dimensions.ai/badge.js"></script> <script type="text/javascript">window.MathJax={tex:{tags:"ams"}};</script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.0/es5/tex-mml-chtml.min.js" integrity="sha256-rjmgmaB99riUNcdlrDtcAiwtLIojSxNyUFdl+Qh+rB4=" crossorigin="anonymous"></script> <script defer src="https://cdnjs.cloudflare.com/polyfill/v3/polyfill.min.js?features=es6" crossorigin="anonymous"></script> <script type="text/javascript">function progressBarSetup(){"max"in document.createElement("progress")?(initializeProgressElement(),$(document).on("scroll",function(){progressBar.attr({value:getCurrentScrollPosition()})}),$(window).on("resize",initializeProgressElement)):(resizeProgressBar(),$(document).on("scroll",resizeProgressBar),$(window).on("resize",resizeProgressBar))}function getCurrentScrollPosition(){return $(window).scrollTop()}function initializeProgressElement(){let e=$("#navbar").outerHeight(!0);$("body").css({"padding-top":e}),$("progress-container").css({"padding-top":e}),progressBar.css({top:e}),progressBar.attr({max:getDistanceToScroll(),value:getCurrentScrollPosition()})}function getDistanceToScroll(){return $(document).height()-$(window).height()}function resizeProgressBar(){progressBar.css({width:getWidthPercentage()+"%"})}function getWidthPercentage(){return getCurrentScrollPosition()/getDistanceToScroll()*100}const progressBar=$("#progress");window.onload=function(){setTimeout(progressBarSetup,50)};</script> </body> </html>